internal/controller/storageclient_controller.go

```diff
@@ -697,6 +702,43 @@ func removeStorageClaimAsOwner(obj client.Object) {
 	}
 }
 
+func (r *storageClientReconcile) reconcileResourcesByGVK(
+	resources [][]byte,
+	obj client.Object,
+	combinedErr error,
+) {
+	retainKeys := make(map[types.NamespacedName]bool, len(resources))
```

Comment:
May I suggest a  more fitting name
```suggestion
	reconciledObjects := make(map[types.NamespacedName]bool, len(resources))
```